appveyor.yml

```diff
@@ -11,28 +13,35 @@ assembly_info:
 
 environment:
   DOTNET_CLI_TELEMETRY_OPTOUT: 1
-  nodejs_version: "8"
+  nodejs_version: "10"
 
 install:
   - git submodule update --init --recursive
-  - ps: Install-Product node $env:nodejs_version
+  - cmd: powershell Install-Product node $env:nodejs_version
+  - sh: nvm install $nodejs_version
+  - sh: sudo apt update
+  - sh: sudo apt install -y libchromaprint-tools
 
 build_script:
-  - C:\msys64\usr\bin\bash -lc "cd \"$APPVEYOR_BUILD_FOLDER\" && exec ./build.sh
+  - cmd: C:\msys64\usr\bin\bash -lc "cd \"$APPVEYOR_BUILD_FOLDER\" && exec ./build.sh
+  - sh: ./build.sh
   
 after_build:
-  - C:\msys64\usr\bin\bash -lc "cd \"$APPVEYOR_BUILD_FOLDER\" && exec ./appveyor-package.sh
+  - cmd: C:\msys64\usr\bin\bash -lc "cd \"$APPVEYOR_BUILD_FOLDER\" && exec ./appveyor-package.sh
+  - sh: chmod +x ./appveyor-package.sh
```

Comment:
Can we change perms on the file to match build.sh and test.sh so this isn't necessary?